views/index.erb

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>99 Bottles</title>
+</head>
+<body>
+  <h1>99 Bottles of Beer</h1>
+<ul class='bottles'>
+  <li><%= @bottles %></li>
+  <li>Take One Down<a href="/:num_bottles">
```

Comment:
the problem with this is the link will go to a path that literally looks like `"/:num_bottles"` rather than a dynamically changing  path.